packages/docs/pages/Datepicker/DatepickerPage.tsx

```diff
@@ -83,6 +81,25 @@ const DatepickerPage = () => (
       }}
       {/* jsx-to-string:end */}
     </CodePreview>
+
+    <H0>Timepicker</H0>
+    <CodePreview>
+      {/* jsx-to-string:start */}
+      {function Example() {
+        const [time, setTime] = useState<string>();
+
+        return (
+          <Form>
+            <FormGroup>
+              <Timepicker locale="en-US" value={time} onTimeChange={(value) => setTime(value)} />
+            </FormGroup>
+          </Form>
+        );
+      }}
+      {/* jsx-to-string:end */}
+    </CodePreview>
+
+    <TimepickerPropTable />
```

Comment:
I would move this into its own page.

Reply:
I was trying to keep the whole Datepicker/Timepicker/Datetimepicker in one page.